tests/cases/standalone/common/view/view.result

```diff
@@ -40,33 +38,33 @@ SELECT x FROM v1 t1(x) WHERE x > 41;
 | 1970-01-01T00:00:00.042 |
 +-------------------------+
 
--- FIXME(dennis): DROP VIEW not supported yet--
--- DROP VIEW v1 --
--- substrait can't process such query currently
--- CREATE VIEW v1 AS SELECT 'whatever';--
--- SELECT * FROM v1; --
--- substrait can't process such query currently
---CREATE OR REPLACE VIEW v1 AS SELECT 42;--
---SELECT * FROM v1;--
 INSERT INTO v1 VALUES (1);
 
 Error: 1004(InvalidArguments), Invalid SQL, error: column count mismatch, columns: 0, values: 1
 
+-- FIXME(dennis): DROP VIEW not supported yet--
```

Comment:
This comment can be removed. Already support `drop view`

Reply:
Already have `drop view` at the end of the test. This statement can be deleted too.